dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassFile.java

```diff
@@ -52,12 +53,20 @@ public boolean equals(Object other) {
     if (other == null || getClass() != other.getClass()) {
       return false;
     }
-    ClassAndJar that = (ClassAndJar) other;
+    ClassFile that = (ClassFile) other;
     return Objects.equals(jar, that.jar) && Objects.equals(className, that.className);
   }
 
   @Override
   public int hashCode() {
     return Objects.hash(jar, className);
   }
+
+  @Override
+  public String toString() {
```

Comment:
What do we need this for?

Reply:
When unit tests fail, having `toString` helps to identify missing objects quickly.

Reply:
Fair enough. I'd still wait to add them until you actually need to rather than adding them by default. It's quick enough to put one in when a unit test is failing and then rerun.